sauna/consumers/base.py

```diff
@@ -12,8 +12,16 @@ def __init__(self, config):
         self.stale_age = config.get('stale_age', 300)
         self.retry_delay = 10
 
+    @property
+    def logger(self):
```

Comment:
At this time we don't have any plugging using log but can you set same property to plugin/base.py ? should be usefull in futur